packable/packable-derive/Cargo.toml

```diff
@@ -15,12 +15,10 @@ proc-macro = true
 
 [dependencies]
 proc-macro-crate = { version = "1.3.1", default-features = false }
```

Comment:
Can you also bump this please? I've tried locally and we're not affected by any breaking change